egs/gop/s5/run.sh

```diff
@@ -0,0 +1,81 @@
+#!/bin/bash
+
+# Copyright 2019 Junbo Zhang
+# Apache 2.0
+
+# This script shows how to calculate Goodness of Pronunciation (GOP) and
+# extract phone-level pronunciation feature for  mispronunciations detection
+# tasks. Read ../README.md or the following paper for details:
+#
+# "Hu et al., Improved mispronunciation detection with deep neural network
+# trained acoustic  models and transfer learning based logistic regression
+# classifiers, 2015."
+
```

Comment:
explain the output in header

Reply:
I explained the format of the output in stage 4.